castle/migrations/0029_challenge.py

```diff
@@ -0,0 +1,31 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
```

Comment:
They update the database. Every time the model gets updated, either to add, delete, or modify columns, Django will automatically generate a file (via python manage.py makemigrations) that will create the migration files for the database. Then you run python manage.py migrate to apply the changes. It's much easier and faster than manually updating each table. There's several of these files with this PR because I made a number of DB changes to make the Challenges module work with the other tables appropriately.